pkg/apic/provisioning/definitions.go

```diff
@@ -129,6 +130,10 @@ type Provisioning interface {
 	CredentialUpdate(CredentialRequest) (RequestStatus, Credential)
 }
 
+type CustomCredential interface {
+	GetIgnoredCredentials() []string
```

Comment:
can we change this to GetIgnoredCredentialTypes ?

Reply:
we are ignoring credentials by name, right? should this be CredentialNames?